src/cljs/lomakkeet/reagent/datepicker.cljs

```diff
@@ -57,15 +67,16 @@
           [:input.form-control
```

Comment:
Ah, the input is already wrapped in `div.input-group` so it should be definitely possible to attach event  handler to that.